pkg/roachprod/install/cluster_synced.go

```diff
@@ -1422,6 +1422,13 @@ func (c *SyncedCluster) Wait(ctx context.Context, l *logger.Logger) error {
 				return res, nil
 			}
 			res.Err = errors.Wrapf(res.Err, "timed out after 5m")
+			logContent, err := c.runCmdOnSingleNode(ctx, nil, node, fmt.Sprintf("tail -n %d %s", 20, vm.StartupLogs), cmdOptsWithDebugDisabled())
+			if err = errors.CombineErrors(err, res.Err); err != nil {
```

Comment:
I realize my diff had this mistake (apologies), but I think the `CombineErrors` here should have been with `err` and `logContent.Err` (not `res`).